yorkie/rpc/server.go

```diff
@@ -34,6 +36,15 @@ import (
 	"github.com/yorkie-team/yorkie/yorkie/rpc/interceptors"
 )
 
+var (
+	// ErrWrongPort occurs when the port in the config is invalid.
+	ErrWrongPort = errors.New("wrong port number for RPC server")
```

Comment:
I think it would be better to use the expression invalid.
```suggestion
	ErrInvalidRPCPort = errors.New("invalid port number for RPC server")
```